GameServer/quests/QuestsMgr/QuestMgr.cs

```diff
@@ -143,7 +143,7 @@ public static GameLiving ResolveLiving(object identifier, GameLiving defaultLivi
 				string tempID = Convert.ToString(identifier);
 
                 // TODO: Dirty Hack this should be done better
-                var mob = DOLDB<Mob>.SelectObject(DB.Column("Mob_ID").IsEqualTo(tempID).Or(DB.Column("Name").IsEqualTo(tempID)));
+                var mob = DOLDB<Mob>.SelectObject(DB.Column("Mob_ID").IsEqualTo(tempID).Or(DB.Column(nameof(Mob.Name)).IsEqualTo(tempID)));

```

Comment:
I was not able to find a nameof for the table's auto-generated primary key. I think this is acceptable however. This kind of record should be retrieved from `FindObjectByKey` instead.